drivers/wireless/ieee80211/bcm43xxx/bcmf_core.c

```diff
@@ -771,10 +772,10 @@ void bcmf_core_reset(FAR struct bcmf_sdio_dev_s *sbus,
   bcmf_write_sbregw(sbus, base + BCMA_RESET_CTL, 0);
   bcmf_read_sbregw(sbus, base + BCMA_RESET_CTL, &value);
 
-  up_udelay(1);
+  nxsig_usleep(1);
```

Comment:
In general I support such changes for ms delays. With us delays we need to be more careful since `nxsig_usleep` rely on systick and that is usually as ms boundary. Another part is that `nxsig_usleep` will work only after scheduler is running, but not on the early init. If all the changes are verified with the HW and timings are sufficient than I do not have any objections.